### PR TITLE
[ja] Localize Stateful Apps into Japanese

### LINKED ISSUE #2821: [ja] Localize Stateful Apps into Japanese
Localize https://glossary.cncf.io/stateful-apps/ into Japanese

### PR DESCRIPTION
**Pull request by Okabe-Junya**
### Describe your changes

Localize Stateful Apps into Japanese

### Related issue number or link (ex: `resolves #issue-number`)

resolves #2821 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by Okabe-Junya** [comment]
The target page on deploy preview:

- https://deploy-preview-2829--cncfglossary.netlify.app/ja/stateful-apps/

**Review by naonishijima — APPROVED** [review]
LGTM

File: content/ja/stateful-apps.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+status: Completed
+category: プロパティ
+tags: ["基礎", "アプリケーション", "プロパティ"]
+---
+
+ステートフル(および[ステートレス](/ja/stateless-apps/))アプリについて話すとき、ステートとは、アプリが設計通りに機能するために必要なあらゆるデータを指します。
+例えば、カートの情報を保存しているオンラインショップなどはステートフルアプリです。
+
+今日、私たちが使用するほとんどのアプリケーションは少なくとも部分的にステートフルです。
+しかし、クラウドネイティブ環境では、ステートフルアプリは課題です。
```
Comment (kaitoii11) [inline content/ja/stateful-apps.md:12]: どちらかというと、難題や難しいというニュアンスかと思います。

**Review by kaitoii11 — CHANGES_REQUESTED** [review]

**Review by kaitoii11 — APPROVED** [review]
LGTM

**Pull request merged**